lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/f2f-hand-off.yaml

```diff
@@ -93,11 +99,41 @@ states:
         targetJourney: FAILED
         targetState: FAILED_NO_TICF
       error:
+        targetState: SORRY_CRI_TECHNICAL_PROBLEM_CRI_F2F
+      temporarily-unavailable:
         targetJourney: TECHNICAL_ERROR
         targetState: ERROR_NO_TICF
-      temporarily-unavailable:
+
+  SORRY_CRI_TECHNICAL_PROBLEM_CRI_F2F:
+    response:
+      type: page
+      pageId: sorry-technical-problem
+      context: f2fCriError
+    events:
+      tryAgain:
+        targetState: CRI_F2F
+      app:
+        targetJourney: TECHNICAL_ERROR
+        targetState: ERROR_NO_TICF
+        checkJourneyContext:
+          mediumConfidence:
+            targetJourney: NEW_P2_IDENTITY
+            targetState: IDENTITY_DOCUMENT_START
+          lowConfidence:
+            targetJourney: NEW_P1_IDENTITY
+            targetState: IDENTITY_DOCUMENT_START
+      webRoute:
         targetJourney: TECHNICAL_ERROR
         targetState: ERROR_NO_TICF
+        checkJourneyContext:
+          mediumConfidence:
+            targetJourney: NEW_P2_IDENTITY
+            targetState: DCMAW_ASYNC_ANOTHER_WAY
+          lowConfidence:
+            targetJourney: NEW_P1_IDENTITY
+            targetState: DCMAW_ASYNC_ANOTHER_WAY
```

Comment:
I think we're also gonna want to run through reset-session here too, especially as the combo of CIC + App VCs can be problematic (for name correlation)